.github/workflows/CI.yml

```diff
@@ -101,3 +101,40 @@ jobs:
       run: python -m pip install . --no-deps
     - name: Run pytest
       run: python -m pytest -ra -q --cov --no-cov-on-fail --cov-report xml
+
+  old-dependencies:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.10']
+        experimental: [false]
+        os: [ubuntu-22.04]
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python ${{ matrix.python-version }}
+
+      uses: actions/setup-python@v5
+    - name: Install system packages
+      run: |
+        sudo apt update
+        sudo apt install libudunits2-dev libgeos-dev libproj-dev proj-data proj-bin
+
+    - name: Setup environment
+      run: |
+        pip install --upgrade pip
+        pip install pdm
+        pdm lock --strategy direct_minimal_versions -v
+        pdm install -v
+        pip install pytest
+        pip freeze
```

Comment:
can this logic be moved to the tox environment for python 3.10?
that way the same test can be run locally and there is no need to change the jobs on CI